docs/QUICKSTART.md

```diff
@@ -0,0 +1,410 @@
+# Quick-Start Guide for uVisor on mbed OS
```

Comment:
It's done at [line 18](https://github.com/ARMmbed/uvisor/pull/272/files#diff-e3d1d3587b83098597a158f42f54b071R18).